packages/excalidraw/element/routing.ts

```diff
@@ -1034,3 +991,30 @@ const getBindPointHeading = (
     elementsMap,
     origPoint,
   );
+
+const getHoveredElements = (
+  origStartGlobalPoint: Point,
+  origEndGlobalPoint: Point,
+  elementsMap: NonDeletedSceneElementsMap | SceneElementsMap,
+) => {
+  // TODO: Might be a performance bottleneck and the Map type
```

Comment:
It's probably fine, though perhaps you could just directly ignore/skip `isDeleted` elements when iterating elements inside `getHoveredElementForBinding`?

Reply:
That doesn't seem like idiomatic, given you can't really "hover" over deleted elements. That NonDeletedSceneElementsMap type is also a guard if you're unfamiliar with the binding code. I'd rather not change that type.

Reply:
Makes sense, let's leave it.